CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Support for the global library path.
```

Comment:
It might be good to reword this so it's a little more obvious to the user what this addition means, e.g. "Unit resolution now includes the global library path".